jar/list_issues.ml

```diff
@@ -0,0 +1,80 @@
+(*
+ * Copyright (c) 2015 David Sheets <sheets@alum.mit.edu>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ *)
+
+open Lwt
+open Cmdliner
+open Printf
+
+module T = Github_t
+
+let ask_github fn = Github.(Monad.run (fn ()))
+
+let string_of_labels labels =
+  let names = List.map (fun { T.label_name } -> label_name) labels in
+  String.concat ", " names
+
+let print_issue user repo issue =
+  let {
+    T.issue_number;
+    issue_title;
+    issue_labels;
+    issue_comments;
+    issue_state;
+    issue_created_at;
+    issue_closed_at;
+  } = issue in
+  printf "%s/%s#%d %s\n" user repo issue_number issue_title;
```

Comment:
Maybe we could do async output in this function. Not that I expect this to be especially useful in the specific example but it would have pedagogic purposes and might make the function more reusable.